src/Bridge/TracerBridge.php

```diff
@@ -28,5 +32,8 @@ public static function getSubscribedEvents(): array
     public function onTerminate(): void
     {
         $this->tracer->flush();
+        if (method_exists($this->tracer, 'reset')) {
```

Comment:
I would propose a bit more explicit (?) ways (alternatives):
1. Check here for the `\Jaeger\Tracer\ResettableInterface` interface with `instanceof` (since there is a separate interface for this in the Jaeger library)
2. Introduce optional dependency in the current class to the `\Jaeger\Tracer\ResettableInterface`
3. Add `kernel.reset` to the `jaeger.tracer` DI service (this is not so explicit way, but it would be more "resistant" to effects of other event subscribers - they can "stop event propagation")

Reply:
Discussed with @allflame.

Let's go with 1+3 variants.

`jaeger.tracer` will have tag `kernel.reset` AND we will call `reset()` over `$this->tracer`, if it implements resettable interface